h/blocklist/views.py

```diff
@@ -0,0 +1,36 @@
+from pyramid import httpexceptions
+from pyramid.view import view_config
+
+from h import models
+from h.api import search as search_lib
```

Comment:
Not sure there's any reason to rename this, is there? It's renamed in `h.api.views` simply because there's already a function in that module called `search`.